Makefile

```diff
@@ -115,6 +115,7 @@ define func_check_folder
 	fi
 endef
 
+test = t/
```

Comment:
This definition doesn't look neat enough. I suggest you define and write at the top of the code:
````
TEST_CASES            ?= t/
```

Reply:
it is a high-frequency operation, I prefer to code more easily.

Reply:
`make test TEST_CASES=t/
make test test=t/`

Reply:
```
make test TEST_CASES=t/
make test test=t/
```

Reply:
I think we should keep the script style

Reply:
`make test CASES=t/`